client/src/Home.js

```diff
@@ -15,14 +15,37 @@ import styles from './css/home.module.css';
 
 function Home(props) {    
 
+    const [data, setData] = useState([]);
+    const [isPending, setIsPending] = useState(true);
+
+    async function getReview(){
+        fetch('https://bookworms-api.vercel.app/api/reviews', {
+            method: 'GET',
+            headers: {'Accept': 'application/json'},
+        })
+        .then(response => {
+            return response.json();
+        })
+        .then(reviews => {
+            let dataArray = []
+            for (var i = 0; i < reviews.data.length; i++) {
+                dataArray.push(reviews.data[i])
+            }
+            setData(dataArray);
+            setIsPending(false);
+        })
+    }
```

Comment:
I haven't figure out how to write this with `async/await`. 
I was writing it as 
`const response = fetch(...., { method: ' GET`, headers : {...}.})`
`const review = response.json()' 
and I got stuck then

Reply:
You are actually pretty close, just need to add await and it should work:
```javascript
async function getReviews() {
  const response = await fetch(...);
  const reviews = await response.json();
  return reviews;
}

// const reviews = await getReviews();

```

Reply:
thank you! i was not expecting an answer yesterday haha

Reply:
how would you write `async/await` for the loop part? can we write it as such or should we keep it with `.then`?

Reply:
Oh, for the for-loop just put it after reviews, when you are using `async/await`, no need to use then.

```javascript
async function getReviews() {
  const response = await fetch(...);
  const reviews = await response.json();
  const dataArray = []
  for (var i = 0; i < reviews.data.length; i++) {
    dataArray.push(reviews.data[i])
  }
  setData(dataArray);
  setIsPending(false);
}

// await getReviews();
```

Reply:
it seemed that we don't need the for-loop just `reviews.data` will do -- I've updated the code according :D